options/locale/locale_en-US.ini

```diff
@@ -957,11 +957,12 @@ confirm_delete_account = Confirm Deletion
 delete_account_title = Delete User Account
 delete_account_desc = Are you sure you want to permanently delete this user account?
 
-email_notifications.enable = Enable Email Notifications
```

Comment:
Is enabled used anywhere else? As disabled is a key, I'm wondering if the matching enabled should be removed.

Reply:
I've traced "enabled" and "andyourown" back to the model now, along with some comments and other references to the old values.